e2e/mock/server.go

```diff
@@ -1,41 +1,59 @@
 package mock
 
 import (
-	"net"
-
-	"github.com/hyperledger/fabric-protos-go/orderer"
-	"github.com/hyperledger/fabric-protos-go/peer"
-	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
 
 type Server struct {
-	GrpcServer *grpc.Server
-	Listener   net.Listener
+	peers   []*Peer
+	orderer *Orderer
 }
 
-func (s *Server) Start() {
+func NewServer(peerN int, Credentials credentials.TransportCredentials) (*Server, error) {
 	blockC := make(chan struct{}, 1000)
```

Comment:
this should be `var txCs []chan struct{}`, right?

Reply:
why? peers should share same chan right?